portfolio/src/main/webapp/script.js

```diff
@@ -192,6 +199,56 @@ function createCommentElement(comment) {
   return commentElement;
 }
 
+/**
+ * Returns a color from red to green as an RGB value, corresponding to the sentimentScore.
+ * @param {Number} sentimentScore - float between [-1.0, 1.0]
+ */
+function getSentimentColor(sentimentScore) {
+  // Initialize the colors
+  const red = {r : 255, g : 0, b : 0};
+  const yellow = {r : 210, g : 210, b : 0};
+  const green = {r : 0, g : 255, b : 0};
```

Comment:
The pure red and green values are very aggressive and often not that easy to read. You could for example pick values from https://clrs.cc/